walletlibrary/src/androidTest/java/com/microsoft/walletlibrary/identifier/HolderIdentifierCreatorTest.kt

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved
```

Comment:
nit: this header:

/**---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

Reply:
for all new files.